libs/execution_context/ebpf_maps.c

```diff
@@ -104,7 +104,7 @@ typedef struct _ebpf_core_lpm_map
     ebpf_core_map_t core_map;
     uint32_t max_prefix;
     // Bitmap of prefix lengths inserted into the map.
-    uint8_t data[1];
+    uint64_t data[1];
```

Comment:
```suggestion
    ebpf_bitmap_t data[1];
```
If you use this suggestion then a bunch of casts can be removed (line 1588, 1610, etc.)

Reply:
The ebpf_bitmap_t type is undefined in this scope.

https://github.com/microsoft/ebpf-for-windows/blob/b6bf2bb6409205bcb59e8d0a04ca61721ee28a61/libs/runtime/ebpf_bitmap.h#L10

It might be possible to make ebpf_bitmap_t a defined type in the header, but it was done this way to abstract the implementation of ebpf_bitmap_t from it's users.